### DISCUSSION TITLE
Stepping down as maintainer (@peterlewis)

### DISCUSSION THREAD

**Discussion by peterlewis** [wg-easy/wg-easy#1175] [Announcements]
Hi folks.

I’m sorry to announce that I’m stepping down as maintainer of wg-easy.

Life is busy and—as a result of this—I’m not able to keep up with the pace of development. Subsequently, I don’t think that I have enough awareness of where things are at, in order to be able to fulfil my responsibilities as a maintainer.

Thank you to @WeeJeWel for creating such a great tool, thank you @pheiduck for your ongoing time and effort contributing to the project and thank you to all contributors for helping to maintain and improve wg-easy

I’ll leave you in the capable and willing hands of @wg-easy/maintainers and look forward to continuing to be a contributing member of the wg-easy project.

**Comment by pheiduck** [comment]
@peterlewis 

It is a pity to hear that you are stepping down as maintainer, you have contributed a lot to the project in your role and I would also like to thank you personally for your support in making decisions.

I wish you all the best for your future!
Best regards
Philip Heiduck
